src/schemas/jsonSchema/index.js

```diff
@@ -1,6 +1,7 @@
 const _ = require('lodash');
 const Ajv = require('ajv').default;
 const traverse = require('json-schema-traverse');
+const { definitions: ucaDefinitions } = require('@identity.com/uca');
```

Comment:
it will use the definition from the uca lib? so if I want to add a new UCA, I need to add in the uca libs and in civic-schema?

Reply:
No. It will actually update the existing UCA definitions as the schema's load - I deliberately tested with v1.0.29 to make sure this works... I will update to 1.0.30 now... on the next UCA release we should remove all the previous schema's